source/_components/media_player.universal.markdown

```diff
@@ -0,0 +1,100 @@
+---
+layout: component
+title: "Universal"
+description: "Instructions how to create a universal media player in Home Assistant."
+date: 2016-01-12 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo:
+ha_category: Media Player
+featured: true
```

Comment:
Please don't feature it as we can only feature a component if we un-feature something else (UI looks weird if there are not 15 featured components). I consider this an advanced component and prefer not to feature it right now.